internal/command/default-command_test.go

```diff
@@ -199,13 +199,16 @@ func TestConditionalVariableRender(t *testing.T) {
 		CmdValidArgsCmd:     nil,
 		CmdRequiredFlags:    nil,
 		CmdFlagValuesCmd:    nil,
-		PkgDir:              "/tmp/test/root",
 	}
 
 	if runtime.GOOS == "windows" {
+		cmd.PkgDir = "\\tmp\\test\\root"
 		assert.Equal(t, fmt.Sprintf("/tmp/test/root/%s/test.bat", runtime.GOOS), cmd.interpolateCmd())
+		assert.Equal(t, "long description - test.exe - test.bat - .exe - .bat", cmd.interpolate(cmd.CmdLongDescription))
```

Comment:
In the current implementation, the interpolation only happens for command related fields:
1. executable
2. args
3. validArgsCmd
4
This line of test (and the one for linux side) is a bit confusing in my point of view, which shows that we interpolate variables in the description as well. 

I do think this line of test is important to test the injection of Binary, Script, Extension, and ScriptExtension variable. But probably better in a different context. WDYT?

Reply:
I created a test for each new field using always the executable to render.